src/streaming.ts

```diff
@@ -124,7 +124,7 @@ export class StreamProcessor {
     return this.eventSource.readyState == StreamProcessor.CONNECTED;
   }
 
-  private stop(): void {
+  stop(): void {
```

Comment:
Any reason you'd want this to be public, but not the equivalent function in `polling.ts`? Just curious

Reply:
Good catch :), I was doing a test and I needed to stop it and then forgot to return to private modifier